### PR TITLE
Modify test folder

### PR DESCRIPTION
**Pull request by meifu**
modify test, use react-scripts test

### CODE REVIEW

**Review by arthurInno — COMMENTED** [review]
Should we put test files near the target code they are testing?

**Review by abc1230429 — APPROVED** [review]

File: jest.config.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 module.exports = {
-  roots: ['<rootDir>'],
+  roots: ['<rootDir>/src/'],
   moduleFileExtensions: ['js'],
   snapshotSerializers: ['enzyme-to-json/serializer'],
```
Comment (arthurInno) [inline jest.config.js]: Do we still need snapshotSerializers when there is another same setting in the package.json file?

**Review by arthurInno — COMMENTED** [review]
Do we still need this when there is another same setting in the package.json file?

File: jest.config.js (hunk 1 above)
Comment (meifu) [reply jest.config.js]: We don't need it here, thanks!

**Review by arthurInno — APPROVED** [review]

**Pull request merged**